cubito/main.cpp

```diff
@@ -320,6 +358,14 @@ void processInput(GLFWwindow* window) {
 
   if (glfwGetKey(window, GLFW_KEY_RIGHT) == GLFW_PRESS)
     camera.ProcessKeyboard(RIGHT, deltaTime);
+  
+  // Solver
+  if (glfwGetKey(window, GLFW_KEY_Z) == GLFW_PRESS) {
+    shuffle_anim = true;

```

Comment:
Pressing Z should not only play the animation, but also generate a new scramble sequence. Right now, I can only scramble once and if I try to scramble twice, it breaks. It should be possible for me to scramble multiple times, even in a row.
```
1. str_scramble = ""
2. Scramble once (str_scramble += GenScramble)
3. Scramble twice (str_scramble += GenScramble)
4. Solve (str_scramble is consumed and now str_scramble = "")
```

Reply:
You would probably have to move `str_scramble` as a global variable.

Reply:
Done manito